source/agora/node/FullNode.d

```diff
@@ -466,4 +486,69 @@ public class FullNode : API
     protected void onAcceptedTransaction () @safe
     {
     }
+
+    /***************************************************************************
+
+        Push the block data to the `block_handlers` target server list
+        set in config.
+
+        Convert block data to JSON serialization and send it POST using Rest.
+
+        Params:
+            block = externalized block data
+
+    ***************************************************************************/
+
+    public void pushBlock (const ref Block block) @safe
+    {
+        foreach (BlockExternalizedHandler handler; block_handlers)
+        {
+            runTask({
+                try
+                {
+                    // API POST: /block_externalized
+                    handler.pushBlock(block);
+                }
+                catch (Exception e)
+                {
+                    log.error(
+                        "Error sending block height #", block.header.height
+                        ," to \"", this.config.node.address, "\": ", e);
+                }
+            });
+        }
+    }
+
+    /***************************************************************************
+
+        Push the preimage to the `block_handlers` target server list
+        set in config.
+
+        Convert `PreImageInfo` to JSON serialization and send it POST using Rest.
+
+        Params:
+            preImage = Received `PreImageInfo`
+
+    ***************************************************************************/
+
+    public void pushPreImage (const ref PreImageInfo preImage) @safe
+    {
+        foreach (PreImageReceivedHandler handler; preimage_handlers)
+        {
+            runTask({
+                try
+                {
+                    // API POST: /preimage_received
+                    handler.pushPreImage(preImage);
+                }
+                catch (Exception e)
+                {
+                    log.error(
+                        "Error sending preImage #",
+                        " (enroll_key: ", preImage.enroll_key, ") to \"",
+                        this.config.node.address, "\": ", e);
```

Comment:
Again wrong destination address.

Reply:
Yes, that's right.
This is the remnant used before adding config.